src/style.css

```diff
@@ -1 +1,83 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
```

Comment:
Avoid using the '*' selector for setting `box-sizing` as it can impact performance. Consider setting `box-sizing` for specific elements or use inheritance from a common parent.